### DISCUSSION TITLE
GIF generator for `model viewer` ?

### DISCUSSION THREAD

**Discussion by shreyasmulay2305** [google/model-viewer#4552] [General]
Do we have any functionality available to take a GIF out of `model-viewer` just like this https://github.com/mrdoob/omggif-example/tree/main

@elalish @mrdoob

**Comment by elalish** [comment]
That would be cool - we provide screenshotting with https://modelviewer.dev/docs/#entrydocs-loading-methods-toBlob so you should be able to do that several times and pack them into an animated GIF in much the same way.

**Reply by shreyasmulay2305** [reply]
Thank you so much @elalish I will try this option.

**Reply by rawwerks** [reply]
@shreyas-aw - did you ever get this working? i had the same thought!
